kedro-training/docs/01_prerequisites.md

```diff
@@ -1,3 +1,57 @@
+# Training pre-requisites
+This Kedro training materials assume the following pre-requisites for your technical understanding.
+
+We assume that you understand these technologies and we won't be teaching how to do any of them during the training. In order to maximise your learning experience from the Kedro training, please review them before the training if you are not familiar with any of them.
+
+- Intermediate Python knowledge
+  - Basic Python
+    - Functions
+    - Classes
+    - Loops
+    - Conditional statements
+    - Common data structures (lists, dictionaries, tuples)
+    - IO operation (reading and writing data)
+  - Importing Python packages using `pip`.
+  - Dependency management with `requirements.txt`
+  - Working on multiple Python files in nested directory structures.
```

Comment:
can we make this list shorter? I don't want to scare off beginners to Kedro :(

Reply:
Agree. I can make `Basic Python` part into one line, but it's not reducing anything... And I think most of them are really necessary to know to follow the training I guess? Happy for any suggestions for making this list look less scary :)